Azure/README.md

```diff
@@ -0,0 +1,37 @@
+# Clear Containers on Azure-VM
+
+This project allows to launch an Azure VM and install all
+stuff in order to run Clear Containers.
+
+# Requirements
+- Install azure-cli V 2.0 https://github.com/Azure/azure-cli
+- Being previously logged in azure with azure client
+
+```bash
+$ az login
+```
+
+## Usage
+
+```bash
+$ sudo bash cc-setup-azure.sh
+```
+
+## Options
+
+| Option | Short description
+| ------ | --------------------------------
+|   -c   | Cloud init file
+|   -d   | Devicemapper disk size
+|   -g   | Resource group
+|   -h   | Help page
+|   -n   | VM name
+|   -o   | OS system
+|   -s   | VM size (e.g. Standard_D2s_v3)
+|   -v   | Show version
+
+## Example
+
+```bash
+$ sudo bash cc-setup-azure.sh -n MyTestVM -g MyTestGroup
+```
```

Comment:
@MarioCarrilloA can you add details on how to login to the VM. The default username for example on azure is not obvious. And where the ssh key is found.